### ISSUE TITLE
lock is not working on form submission 

### ISSUE THREAD

**Issue by sahoomonalisa** [auth0/lock#1477]
Facing two issue on lock on production app : 

-  onSubmit (pressing enter) is not working with closable: true. It immediately closes the lock.

- With closable: false , on otp screen onsubmit is taking back to email screen while on submitting (pressing enter) form with clicking button is working fine.

- [ ] Code snippet or sample project that reproduces the bug

  ```var options = {
    allowAutocomplete: true,
    passwordlessMethod: 'code',
    allowedConnections: ['email'],
    theme: {
      logo: logo,
      primaryColor: '#2096c8',
      labeledSubmitButton: false
    },
    auth: {
      params: {
        scope: 'openid email profile'
      },
      redirect: false,
      responseType: 'token',
      callbackURL: BASE_URL,
      redirectUrl: BASE_URL
    },
    languageDictionary: {
      title: 'isoshealth'
    },
    closable: false,
    autoclose: true,
    callbackURL: BASE_URL,
    configurationBaseUrl: configurationBaseUrl
  };

  const lock = new Auth0LockPasswordless(clientId, domain, options);

- [ ] Lock version : 11.9.0

- [ ] Browser & OS : chrome(Version 64.0.3282.167 (Official Build) (64-bit)) , linux

Please resolve the above issue or suggest solution as it is coming on production app.

Note : This bug is coming on production env only while on localhost its working fine.

**Comment by miguelcobain** [comment]
Can confirm this is happening on lock `11.9.0`.

After I forced the downgrade with (using yarn)

```json
"resolutions": {
  "auth0-lock": "11.8.1"
}
```

it started working again.

**Comment by sahoomonalisa** [comment]
@miguelcobain  It didn't work even after changing version . I have tried this with npm as i am using npm in my project.

**Comment by b-reklass** [comment]
I can confirm @miguelcobain fix of reverting to 11.8.1 is working for me, same issue with the latest in production environment only.

**Comment by luisrudge** [comment]
Thanks everyone for the investigation and the workaround. I'll see if we can come up with a patch fix on Monday.

**Comment by luisrudge** [comment]
@sahoomonalisa you have to downgrade to 11.8.1. How are you doing that?

This is the command: `npm install -S auth0-lock@11.8.1`

**Comment by andrew-me** [comment]
Hopefully this fixes the issues: https://github.com/auth0/lock/pull/1478

**Comment by luisrudge** [comment]
fixed by #1478
